src/ess/livedata/dashboard/widgets/correlation_histogram_widget.py

```diff
@@ -0,0 +1,118 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2025 Scipp contributors (https://github.com/scipp)
+from __future__ import annotations
+
+from typing import Any
+
+import pandas as pd
+import panel as pn
+
+from ..correlation_histogram import CorrelationHistogramController
+from .configuration_widget import ConfigurationModal
+
+
+class CorrelationHistogramWidget:
```

Comment:
As with other widgets, this is preliminary. There are now three widgets listing jobs and/or their data in different ways, and we need to figure out if and how they should be unified, but that is beyond the scope of this PR.